src/sentry/static/sentry/app/views/organizationIntegrations/integrationListDirectory.tsx

```diff
@@ -520,6 +520,16 @@ export class IntegrationListDirectory extends AsyncComponent<
                   {tct('No Integrations found for "[searchTerm]".', {
                     searchTerm: searchInput,
                   })}
+                  <EmptyResultsBodyBold>
+                    {t("Not seeing what you're looking for?")}
```

Comment:
Phillip just spoke to Molly. Can we change this line?
```suggestion
                    {t("Don't see what you're looking for?")}
```